lib/jsonapi/authorization/authorizing_processor.rb

```diff
@@ -219,6 +219,10 @@ def authorize_remove_to_many_relationships
 
         related_records = related_resources.map(&:_model)
 
+        if related_records.count != params[:associated_keys].uniq.count
```

Comment:
Is there any way `.count` would accidentally trigger an SQL `COUNT` query even if a cached version would exist in these changes? I.e. should we use `.size` instead of `.count` here? Does it matter?

Reply:
Since `related_records` is created by calling `map` on a collection of Resources, it will always be an array and not trigger any SQL queries. On the other hand, I forgot that `Array#size` is more performant than `count` so I'll change that here.